edb/common/ast/codegen.py

```diff
@@ -52,6 +52,22 @@ def node_visit(self, node: base.AST) -> None:
         visitor = getattr(self, method, self.generic_visit)
         return visitor(node)
 
+    def visit_indented(
+        self, node: base.AST, indent: bool = False, nest: bool = False
```

Comment:
Should indent be true by default?

Reply:
Ah, yes. I wanted to just override `visit`, but typing complained extensively.